### PR TITLE
Yb 11 product detail

### PR DESCRIPTION
**Pull request by yelab07**
## Changes
_List Changes Introduced by this PR_
1. Product detail page built according to the design 

## Purpose
The purpose was to build the pages with the product details coming from the API in a way it is attractive  to the user  with included information that would impress the user to go ahead and  add the product in the cart .

## Learning
For the first time I used react-native-size-matters a react native utility  to help me scale my styling to different screen sizes
 [react-native-size-matters ](https://www.npmjs.com/package/react-native-size-matters-ch)

### CODE REVIEW

File: screens/Home/ProductDetailsScreen.tsx [hunk 1]
```diff
@@ -1,124 +1,142 @@
...
+const ProductDetailsScreen = ({ onPress }: Props) => {
   const navigation = useNavigation();
-  const route = useRoute<RouteProp<ParamList, "Params">>()
-  const { id } = route.params
+  const route = useRoute<RouteProp<ParamList, "Params">>();
+  const { id } = route.params;
   const { data } = useProductQuery(id);
+  const [quantity, setQuantity] = useState<any>(1);
 
+  const onAddToCart = () => { };
```
Comment (eangele1) [inline screens/Home/ProductDetailsScreen.tsx:29]: Great work! Seems all good to me. Looking forward to the next part.

**Review by eangele1 — APPROVED** [review]

File: screens/Home/ProductDetailsScreen.tsx (hunk 1 above)
Comment (yelab07) [reply screens/Home/ProductDetailsScreen.tsx:29]: Thank you Ezer!

**Comment by kmausolf** [comment]
I don't think we should delete the yarn.lock file.  It would be best to undo the delete, even if you're not using yarn at this time.

**Comment by yelab07** [comment]
Thanks @kmausolf for your feedback I have got the Yarn file back !

**Pull request merged**